packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -776,6 +873,7 @@ class Viewport implements IViewport {
       viewAngle,
       flipHorizontal,
       flipVertical,
+      resetOffsets,
```

Comment:
`resetOffsets` should not be on camera interface. It can be the second argument of `setCamera` though (`setCamera(cameraInterface, resetOffset)`) but again the `resetOffset` is vauge and not what it actually does (which is `set` initial camera)

Reply:
OK, will make it the second argument